src/createServer.js

```diff
@@ -1,3 +1,39 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('../src/convertToCase/convertToCase');
+const { validation } = require('./validation');
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const { pathname, searchParams } = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = pathname.slice(1);
+    const targetCase = searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
```

Comment:
Sending 'application/json' as Content-Type header before validation may result in unnecessary overhead in case of errors. Consider moving this line after validation, immediately before the response is sent.